crates/hash/src/hash.rs

```diff
@@ -62,6 +63,12 @@ impl AsRef<[u8]> for Inner {
     }
 }
 
+impl Debug for Inner {
```

Comment:
Why not `derive(Debug)` on `Inner` instead? (L36 https://github.com/profianinc/drawbridge/pull/38/files#diff-7ff10f8cc2f21abbd9822afaa5c33639a819e98ac9cdc9758c037e2f238ef853R36)

Reply:
Because `Inner` contains objects from the `sha2` crate, which don't also derive Debug, so it's a compiler error.

Reply:
Let's document that